<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUserSetControl</class>
 <widget class="QDialog" name="CUserSetControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserSetControl</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/logo.png</normaloff>resources/logo.png</iconset>
  </property>
  <widget class="QGroupBox" name="UserSetControl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>UserSetControl</string>
   </property>
   <widget class="QComboBox" name="UserSetSelector">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="UserSetSelectorLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>26</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UserSetSelector</string>
    </property>
   </widget>
   <widget class="QLabel" name="UserSetSaveLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>86</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UserSetSave</string>
    </property>
   </widget>
   <widget class="QComboBox" name="UserSetDefault">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="UserSetDefaultLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>116</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UserSetDefault</string>
    </property>
   </widget>
   <widget class="QLabel" name="UserSetLoadLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>56</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UserSetLoad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UserSetLoad">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>UserSetLoad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UserSetSave">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>UserSetSave</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="UserSet_Close">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>UserSetSelector</tabstop>
  <tabstop>UserSetLoad</tabstop>
  <tabstop>UserSetSave</tabstop>
  <tabstop>UserSetDefault</tabstop>
  <tabstop>UserSet_Close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
